.github/workflows/postman-docker.yaml

```diff
@@ -0,0 +1,62 @@
+# @author Jeff Carpenter
+name: Stargate Postman Collections
+
+# runs on
+# * manual trigger
+on:
+  workflow_dispatch:
+
+jobs:
+
+  # Runs Stargate Postman Collections against local Stargate instance using docker compose scripts
+  automated-api-tests:
+    runs-on: ubuntu-latest
+    strategy:
+
+      # let all tests run, can find multiple failures in different apis
+      fail-fast: false
+
+      # props:
+      # backend - cassandra backend to run
+      # collection - Postman collection to run
+
+      matrix:
+        backend: [ cassandra-40, cassandra-311, dse-68 ]
+        collection: [ docsapi-library, graphqlapi-library, restapi-users ]
+        include:
+          - backend: cassandra-40
+            path: docker-compose/cassandra-4.0
+          - backend: cassandra-311
+            path: docker-compose/cassandra-3.11
+          - backend: dse-68
+            path: docker-compose/dse-6.8
+          - collection: docsapi-library
+            collection-id: 12949543-caba2a02-6559-486a-9e4a-d5c0791fd296
+            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+          - collection: graphqlapi-library
+            collection-id: 17930693-65da5c64-561a-449b-a0e8-0318575f6871
+            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+          - collection: restapi-users
+            collection-id: 17930693-47ab5f0d-407e-48cf-aa11-c51d129f1eef
+            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Postman CLI
+        run: |
+          curl -o- "https://dl-cli.pstmn.io/install/linux64.sh" | sh
+      - name: Login to Postman CLI
+        run: postman login --with-api-key ${{ secrets.POSTMAN_API_KEY }}
```

Comment:
wha about using a dedicated action: https://github.com/marketplace/actions/newman-cli-postman-action

Reply:
idk, this step actually was autogenerated for me by postman.